src/tools/pub/cmd/index.md

```diff
@@ -30,7 +30,7 @@ on the [Flutter website]({{site.flutter}}).
   Although you might still find examples of
   using the standalone `pub` command instead of
   `dart pub` or `flutter pub`,
-  the standalone `pub` command is deprecated.
+  the standalone `pub` command has been removed.
```

Comment:
**comment:** This runs a bit long, but it helps as is. We should return to tighten up the language here.